### PR TITLE
Scala 2.10

### PR DESCRIPTION
**Pull request by adamdec**
Added support to parse JSON schema directly from provided file content    60c500a  
Changed a method name which loads test resources

### CODE REVIEW

**Comment by adamdec** [comment]
- Added support to parse JSON schema directly from provided file content    60c500a  
- Changed a method name which loads test resources

File: build.sbt [hunk 1]
```diff
@@ -32,16 +32,21 @@ pomExtra := (
...
+        <email>patrick.baier@zalando.de</email>
+        <url>https://github.com/zalando</url>
+      </developer>
+      <developer>
+        <name>Adam Dec</name>
+        <email>adamdec85@gmail.com</email>
+        <url>https://github.com/adamdec</url>
+      </developer>
+    </developers>
+  )
```
Comment (hesserp) [inline build.sbt]: please add newline at end of file

File: src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala [hunk 2]
```diff
@@ -121,4 +126,4 @@ object SchemaConverter {
     val content = contentPath.asSingleJson(json).as[JsObject]
     convertJsonStruct(new StructType, content, content.keys.toList)
   }
-}
+}
```
Comment (hesserp) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: please add newline at end of file

File: src/test/scala/org/zalando/spark/jsonschema/SchemaConverterTest.scala [hunk 3]
```diff
@@ -52,14 +57,18 @@ class SchemaConverterTest extends FunSuite {
...
     assert(dbWithSchema.schema === schema)
     assert(dbWithSchema.select("name").collect()(0)(0) === "aaa")
     assert(dbWithSchema.select("address.zip").collect()(0)(0) === null)
-    intercept[AnalysisException] { dbWithSchema.select("foo") }
+    intercept[AnalysisException] {
+      dbWithSchema.select("foo")
+    }
   }
-}
+}
```
Comment (hesserp) [inline src/test/scala/org/zalando/spark/jsonschema/SchemaConverterTest.scala]: please add newline at end of file

File: src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala [hunk 4]
```diff
@@ -12,5 +14,11 @@ object SparkTestEnv {
...
-  lazy val sqlContext: SQLContext = new org.apache.spark.sql.SQLContext(sparkContext)
-}
+  lazy val sqlContext: SQLContext = new SQLContext(sparkContext)
+
+  def getTestResourceContent(relativePath: String): String = {
+    val relPath = getClass.getResource(relativePath)
+    require(relPath != null, s"Path can not be reached: $relativePath")
+    Source.fromURL(relPath).mkString
+  }
+}
```
Comment (hesserp) [inline src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala]: please add newline at end of file

**Comment by hesserp** [comment]
:+1:

File: build.sbt (hunk 1 above)
Comment (adamdec) [inline build.sbt]: Done

**Comment by hesserp** [comment]
:+1:

**Pull request merged**